docs/changehistory/Roadmap.md

```diff
@@ -1,143 +1,138 @@
+- [X] Support for creating multi-process connectors using iTwin.js backends. One process links with source application api to read the source application files. It then communicates with another iTwin.js process via IPC to update the iModel.
 
 ### iModel Editing applications
 
-- Support interactive iModel creation, editing, and augmenting on desktops via Electron.
+- [X] Support interactive iModel creation, editing, and augmenting on desktops via Electron.
 
 ### Mobile Device Support
 
-- Minimize memory, power, and bandwidth requirements for mobile browsers.
-- Improve usability for touch and small screen form factors.
-- Support both *completely offline* and *occasionally connected* tablets and phones (iOS and Android).
+- [ ] Minimize memory, power, and bandwidth requirements for mobile browsers.
+- [X] Improve usability for touch and small screen form factors.
+- [ ] Support both *completely offline* and *occasionally connected* tablets and phones (iOS and Android).
 
 ### Non-Graphical Data Presentation
 
-- Filtering presentation rules and rulesets by supported ECSchema version.
-- Auto-updating data in all presentation rules -driven components when:
-  - data in source iModel changes
-  - presentation rules change
-  - presentation ruleset variables change
-
-#### Tree-related Improvements
-
-- Making tree nodes "favorite" to make them appear at the top of the hierarchy.
-- Improve filtering performance.
-- Excluding or exclusively including some nodes from the hierarchy when filtering.
+- [x] Filtering presentation rules and rulesets by supported ECSchema version.
+- [x] Auto-updating data in all presentation rules -driven components when:
+  - [x] data in source iModel changes
+  - [x] presentation rules change
+  - [x] presentation ruleset variables change
 
 #### Properties-related Improvements
 
-- Nested property grouping.
-- Clickable navigation properties that select the target element.
-- Better support for composite properties (arrays, structs, points).
-- Support relationship properties.
-- Support ad-hoc properties.
+- [x] Nested property grouping.
+- [x] Clickable navigation properties that select the target element.
+- [ ] Better support for composite properties (arrays, structs, points).
+- [ ] Support relationship properties.
+- [ ] Support ad-hoc properties.
 
 ### User Interface
 
-- Extend functionality of panel-based AppUi.
-- Improve touch support and user experience on mobile devices.
-- Multi-window support for desktop Electron applications.
-- Enhance accessibility and keyboard navigation.
-- Enhance UiItemProvider to support:
-  - More type editors, especially those necessary to read/write workflows
-  - Async list population
-- Provide an API to allow apps, extensions, and packages to supply settings to the Settings Stage.
+- [x] Extend functionality of panel-based AppUi.
+- [ ] Improve touch support and user experience on mobile devices.
```

Comment:
Since iOS/Android don't use AppUi, is this on their roadmap or left from when we thought we needed to provide it?

Reply:
MobileUI is available for use, I think we can mark this done.

Reply:
You'll need to do so in a separate PR; this one was already merged.
"Mobile Device Support" already had this, which is marked done:
`[X] Improve usability for touch and small screen form factors.`

Reply:
I wouldn't want to mark it as done yet. We're not out of beta.... Nothing in this should be checked as done if it is still in an EA or non-public status.